modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -351,7 +351,7 @@
 /obj/item/clothing/suit/space/hardsuit/deathsquad/praetor
 	name = "Praetor Suit"
 	desc = "And those that tasted the bite of his sword named him... The Doom Slayer."
-	armor = list("melee" = 75, "bullet" = 55, "laser" = 55, "energy" = 45, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 75, "bullet" = 40, "laser" = 40, "energy" = 35, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
```

Comment:
armor = list("melee" = 55, "bullet" = 30, "laser" = 20, "energy" = 15, "bomb" = 60, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)

Reply:
This is my suggestion, as this thing is way too fucking powerful even with the nerf.

Reply:
No lmao that is literally worse than other megafauna armor.